driver/normalizer/annotation.go

```diff
@@ -236,10 +247,27 @@ var AnnotationRules = On(Any).Self(
 		On(HasInternalType(pyast.Return)).Roles(Return),
 		On(HasInternalType(pyast.Break)).Roles(Break),
 		On(HasInternalType(pyast.Continue)).Roles(Continue),
+		// FIXME: IfCondition bodies in Python take the form:
+		// 1 < a < 10
+		// - left (internalRole): 1 (first element)
+		// - Compare.ops (internalType): [LessThan, LessThan]
+		// - Compare.comparators (internalType): ['a', 10]
+		// The current mapping is:
+		// - left: BinaryExpressionLeft
+		// - Compare.ops: BinaryExpressionOp
+		// - Compare.comparators: BinaryExpressionRight
```

Comment:
Yes, they're usually called LHS and RHS. Still that info could be implicit, not explicit, without loss of semantic information and reducing Roles pollution.